CLAUDE.md

```diff
@@ -0,0 +1,23 @@
+# PocketSizeFund Assistant Memory
+
+## GitHub Issue Creation
+
+When asked to create a new GitHub issue:
+
+1. Follow the structure in `.github/ISSUE_TEMPLATE.md`
+2. Include:
+   - A descriptive title (without [FEATURE|BUG] prefix)
+   - Add the "backlog" label and either "bug" or "feature" label as appropriate
+   - Explain why the feature is needed
+   - Detail the benefits it will bring
+   - Propose two implementation options (after considering at least five)
```

Comment:
I like this a lot. It's not a huge thing but I'd love it if it still adhered to the templates in the `ISSUE_TEMPLATE` directory.

Reply:
done
![CleanShot 2025-05-14 at 09 54 36](https://github.com/user-attachments/assets/15ff527c-1b98-4e15-a0f5-5bc5ba19cc15)